installer/windows/build_installer.py

```diff
@@ -289,14 +289,6 @@ def create_wxs_file():
             Impersonate="no"
             Return="check"
         />
-        <CustomAction
-            Id="RollbackAcrylicInstall"
-            Directory="AcrylicDNSProxy"
-            ExeCommand="cmd /c &quot;[AcrylicDNSProxy]UninstallAcrylicService.bat&quot;"
-            Execute="rollback"
-            Impersonate="no"
-            Return="ignore"
```

Comment:
IMO, we should keep this since there is a high probability that we will reuse it later

And since it has been removed from the `InstallExecuteSequence`, it won't be run

Reply:
yeah but I don't like to let dead code